ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -85,11 +85,13 @@ class BottomSheetDemoController: UIViewController {
         let sheetContentView = UIView()
 
         let secondarySheetController = BottomSheetController(expandedContentView: sheetContentView)
+        secondarySheetController.delegate = self
         secondarySheetController.collapsedContentHeight = 250
         secondarySheetController.isHidden = true
         secondarySheetController.shouldAlwaysFillWidth = false
         secondarySheetController.shouldHideCollapsedContent = false
         secondarySheetController.isFlexibleHeight = true
+        secondarySheetController.allowsSwipeToHide = true
```

Comment:
we should either have a toggle or clear description what is the behavior of this seconarydsheetcontroller in the demo app itself for future readers.

Reply:
These are the sheets that show when you tap the "show transient sheet" button in the demo app. I added an explanation in a comment.